systemtests/tests/commandline-options/testrunner

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+
+TestName="$(basename "$(pwd)")"
+export TestName
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+"${rscripts}"/cleanup
+"${rscripts}"/setup
+
+
+# Fill ${BackupDirectory} with data.
+setup_data
+
+# With this test we try to summon some of the forgotten options that are not
+# used in other systemtests.
+
+jobname=backup-bareos-fd
+
+start_test
+
+#Testing that -u and -g do not cause problems
+${rscripts}/bareos-ctl-dir start "-u randomuser -g randomgroup -d 100"
+${rscripts}/bareos-ctl-dir stop
+
+
+#Testing the creation and deletion of pid files with -p
+pid_file="${tmp}"/pid_file.out
+
+# Director
+
+${rscripts}/bareos-ctl-dir start "-p ${pid_file} -d 100" && sleep 5
+
+if [ ! -s ${pid_file} ]; then
+    echo "Pid file was not written."
+    estat=1
```

Comment:
Setting the estat to an individual value for each error could serve to quicker understand what exactly failed